code/parachain/runtime/dali/src/lib.rs

```diff
@@ -1270,8 +1436,13 @@ construct_runtime!(
 
 		CallFilter: call_filter = 140,
 
-	  // Cosmwasm support
-	  Cosmwasm: cosmwasm = 180
+		// Cosmwasm support
+		Cosmwasm: cosmwasm = 180,
+
+		// IBC support
+		IbcPing: pallet_ibc_ping = 190,
+		// pallet-ibc, should be the last module in your runtime
```

Comment:
why? what if there is other module which will want to be last? from what this requirements steams?

Reply:
may we can make it 200 and tell which modules can be after and before? please guide what should be here and why.

Reply:
also, would beffy migration happen in place?

Reply:
so sure 2 modules cannot be both last

Reply:
actually, you can put it anywhere

Reply:
Then @vmarkushin let's remove the comment on it needing to be last.